src/resources/views/fields/number.blade.php

```diff
@@ -9,7 +9,7 @@
         	type="number"
         	name="{{ $field['name'] }}"
             id="{{ $field['name'] }}"
-            value="{{ old($field['name']) ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : '' )) }}"
+            value="{{ old($field['name']) !== null ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : '' )) }}"
```

Comment:
`!is_null(...)` seems more elegant unless I'm missing something here.

In fact that's such a ternary operator mess that I'd be inclined to refactor this some other way.

Reply:
Totally agree with you here @lloy0076 . Also agree with @abarta that this is a problem. So I have something very nice for you guys :-) Null coalesce operator will fix both your issues :-)

So:
```diff
-{{ old($field['name']) !== null ? old($field['name']) : (isset($field['value']) ? $field['value'] : (isset($field['default']) ? $field['default'] : '' )) }}
+{{ old($field['name']) ?? $field['value'] ?? $field['default'] ?? '' }}
```

Pretty sexy, right? :-)

Reply:
Gotta love PHP 7 for this :-)